### PR TITLE
Split NewsManager by Sources

### PR DESCRIPTION
**Pull request by mathiasquintero**
### Issue
Fixes: #43 
Fixes: #210 (At least the API part)

### CODE REVIEW

File: TUM Campus App/News.swift [hunk 1]
```diff
@@ -11,14 +11,21 @@ import Sweeft
 
 final class News: DataElement {
     
+    enum Source: String {
+        case movie = "2"
+        case regular = "1"
```
Comment (kordianbruck) [inline TUM Campus App/News.swift]: Source is a primary key in the table of the newsSources. Its currently right, that the ID 2 is the TU Film, but the ID can be anything. You should account for that as we add new sources in the future.

Here are all sources: 
screenshot

Comment (mathiasquintero) [reply TUM Campus App/News.swift]: Ok. So do we expect the source ids to change over time?

File: TUM Campus App/NewsSource.swift [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    static let operatingSystems = News.Source(identifier: 6)
+    static let newsspreadFMI = News.Source(identifier: 7)
+    static let newsspreadChemistry = News.Source(identifier: 8)
+    static let newsspreadSG = News.Source(identifier: 9)
+    static let newsspreadWI = News.Source(identifier: 10)
+    static let newsspreadPhysics = News.Source(identifier: 11)
+    static let newsspreadLab = News.Source(identifier: 12)
+    static let newsspreadWeihenstephan = News.Source(identifier: 13)
+    static let alumni = News.Source(identifier: 14)
+    static let impulsiv = News.Source(identifier: 15)
```
Comment (kordianbruck) [inline TUM Campus App/NewsSource.swift]: Thats more like it :heart:

Comment (mathiasquintero) [reply TUM Campus App/NewsSource.swift]: I also love bitmasks

File: TUM Campus App/NewsSource.swift [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+        .newsspreadChemistry,
+        .newsspreadSG,
+        .newsspreadWI,
+        .newsspreadPhysics,
+        .newsspreadLab,
+        .newsspreadWeihenstephan,
+        ]
+    
+    static var restOfNews: News.Source {
+        return News.Source.all.subtracting([.movies, .newsSpread])
```
Comment (mathiasquintero) [inline TUM Campus App/NewsSource.swift]: I hope you're cool with this. Everything that is not in movies or any of the newsSpred sources falls here

Comment (kordianbruck) [reply TUM Campus App/NewsSource.swift]: Yea, thats probably the best way to handle it.

**Pull request merged**